R/import-gr-skey-tif-to-pg-rast.R

```diff
@@ -36,6 +42,7 @@ import_gr_skey_tif_to_pg_rast <- function(
 
   gr_skey_rast <- crop_list[[1]]
   make_rast <- crop_list[[2]]
+  make_rast <- terra::extend(make_rast,gr_skey_rast )
```

Comment:
What is this for?

Reply:
edge case where the extent is bigger then extent of the mask TIF.  It fills in the extra cells with no data